cmd/query/app/apiv3/grpc_gateway_test.go

```diff
@@ -60,6 +61,9 @@ const (
 //	REGENERATE_SNAPSHOTS=true go test -v ./cmd/query/app/apiv3/...
 var regenerateSnapshots = os.Getenv("REGENERATE_SNAPSHOTS") == "true"
 
+// The tests in http_gateway_test.go set this to true to use manual gateway implementation.
+var useHTTPGateway = false
```

Comment:
not the cleanest way, but hopefully we'll just delete grpc-gateway based implementation and simplify/consolidate the tests